src/vmm/src/guest_config/aarch64/mod.rs

```diff
@@ -0,0 +1,12 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::collections::HashMap;
+
+/// CPU configuration for aarch64 CPUs
+pub struct Aarch64CpuConfiguration {
+    /// Register values as a key pair
+    /// Key: Register pointer
+    /// Value: Register value
+    pub regs: HashMap<u64, u128>,
```

Comment:
There will be eventually. To make use of the template using bit masks to modify values. the host CPU configuration will need to be looked up so that the template can be applied to something. The process of applying the masks will be keyed as well so they can look up the register in this configuration map directly.

Reply:
Ok, that makes sense indeed.